src/utils/dataset_db/db.py

```diff
@@ -5,7 +5,7 @@
 import pymongo
 
 
-def get_collection() -> pymongo.collection.Collection:
+def get_collection(name="organizations") -> pymongo.collection.Collection:
```

Comment:
maybe we could change this so there's no default collection anymore since it seems like we'll be using many collections which have all have somewhat equal utility

Reply:
or down the road change it to be the fully clean, most useful dataset

Reply:
good point. done and done

Reply:
only problem is now one of the circleci tests is failing because of this change, probably relies on the default argument